po/pt.po

```diff
@@ -829,7 +829,7 @@ msgstr "Selecionar"
 #: src/modules/quickaccess/e_mod_quickaccess.c:381
 #: src/modules/shot/e_mod_main.c:316 src/modules/shot/e_mod_main.c:469
 #: src/modules/shot/e_mod_main.c:732 src/modules/shot/e_mod_main.c:758
-#: src/modules/shot/e_mod_main.c:998 src/modules/xkbswitch/e_mod_config.c:529
+#: src/modules/shot/e_mod_main.c:998 src/modules/xkbswitch/e_mod_config.c:559
 msgid "Cancel"
 msgstr "Cancelar"
 
```

Comment:
this should happen? Just updated from the .pot file!

Reply:
hmm, not sure. I need to compile, switch to portugese and test if OK

Reply:
Ok.